dashboard/settings.py

```diff
@@ -288,10 +288,10 @@
     },
 }
 RANDOM_PASSWORD_DEFAULT_LENGTH = ENV.get('RANDOM_PASSWORD_DEFAULT_LENGTH', 32)
-DB_CACHE_CONFIGS = ENV.get('DB_CACHE_CONFIGS', {"CACHE_TTL": 7200, "BACKEND": "django_mysql.cache.MySQLCache",
-                                                "LOCATION": "django_myla_cache",
-                                                "CACHE_KEY_PREFIX": "myla",
-                                                "CACHE_OPTIONS": {"COMPRESS_MIN_LENGTH": 5000, "COMPRESS_LEVEL": 6}
+DB_CACHE_CONFIGS = ENV.get('DB_CACHE_CONFIGS', {'CACHE_TTL': 7200, 'BACKEND': 'django_mysql.cache.MySQLCach',
```

Comment:
Typo in the module name, `MySQLCache`.  I think this would crash if run as it is.

```suggestion
DB_CACHE_CONFIGS = ENV.get('DB_CACHE_CONFIGS', {'CACHE_TTL': 7200, 'BACKEND': 'django_mysql.cache.MySQLCache',
```

I really don't like the popular Python style of using strings to refer to modules or other code symbols.    This typo demonstrates why.  The developer's IDE or the Python interpreter didn't catch this problem because the name of the module is in a string.

I think it'd be better if this could be done using code like `django_mysql.cache.MySQLCache.__name__` if a string is required, or just `django_mysql.cache.MySQLCache` if a reference to the module is acceptable.  Each of these may require importing the module first.

Reply:
Yeah, that only gets you the name of the module though "MySQLCache". 

It looks like this is a "Django-ism" and [they have a pattern for this.](https://stackoverflow.com/a/33627620/3708872)

```
from django.utils.module_loading import import_string
my_module = import_string('django_mysql.cache.MySQLCache')
```

If it's successful my_module just has the name of the module, if not it throws an ImportError. So we *could* import this package and replace this string with `import_string('django_mysql.cache.MySQLCache')` for safety.

Reply:
And yeah, I was finishing up and reviewing some other changes today but I have this all checked out and ready to go for tomorrow morning.

Reply:
I forgot Python's `__name__` just gives the name of the module (or the "leaf node"), not the full hierarchy.

What I'm suggesting is the opposite of Django's `import_string()`.  Import a module so that it can be checked by the IDE AND will throw exceptions at runtime if it's not found.  When the module is imported, have a way to get its fully qualified name as a string to use in places where it's needed.

Or maybe an approach could be to import the module and reference it instead of putting its name in a string.  I'm not sure whether Django (or the DB module) supports that, but it would be a shame if it didn't.

Reply:
> If you look at the setting.py all modules are defined as strings, in INSTALLED_APPS, MIDDLEWARE, DATABASES.

Yeah, I hate that, too.  😆  I suppose they probably have a good reason for doing it this way.  🤞 

> I think this is just a convenience to avoid having to import every module in the settings. This really isn't any different from those.

Maybe this isn't the place to discuss, but why would importing the modules be an **_in_**-convenience?

> I think we just have to fix typos as they will come up in these cases and don't really see a need to break from the pre-existing patterns.

I begrudgingly agree that it's probably not the time to challenge established Django patterns. ¯\_(ツ)_/¯